reference-architectures/partner-network-connect-aws/test/go.mod

```diff
@@ -0,0 +1,133 @@
+module github.com/digitalocean/scale-with-simplicity/reference-architectures/partner-network-connect-aws/test
+
+go 1.24.2
+
+require (
+	github.com/digitalocean/godo v1.155.0
+	github.com/digitalocean/scale-with-simplicity/test v0.0.0
+	github.com/gruntwork-io/terratest v0.50.0
+	github.com/stretchr/testify v1.10.0
+)
+
+replace github.com/digitalocean/scale-with-simplicity/test => ../../../test
```

Comment:
Just to confirm, this will work on upstream right?

Reply:
Never mind bad question from my side, I just asked because we were using replace with local path here.